src/_help/continuous-integration/github-actions.md

```diff
@@ -15,7 +15,7 @@ Bump.sh keeps your API docs always synchronized with your codebase. With this [G
 
 Start with creating a documentation on [Bump.sh](https://bump.sh).
 
-Once you've set up an API Documentation, go to Settings > CI Deployment, copy the access token, then add it to GitHub Settings > Secrets > Actions as a new environment variable called `BUMP_TOKEN`.
+Once you've set up an API Documentation, go to Settings > CI Deployment, copy the access token, then add it to GitHub Settings > Secrets > Actions as a new secret called `BUMP_TOKEN`.
```

Comment:
They even seem to call them “Environment secrets” now

![Capture d’écran du 2024-01-08 18-29-15](https://github.com/bump-sh/docs/assets/904193/9153df75-6746-4ecb-be78-4bc4a85937ac)

```suggestion
Once you've set up an API Documentation, go to Settings > CI Deployment, copy the access token, then add it to GitHub Settings > Secrets > Actions as a new environment secret called `BUMP_TOKEN`.
```